.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
```

Comment:
:red_circle: [The project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) states that there should be ` test.md ` in this .gitignore file. Kindly add it

![Screenshot from 2022-12-03 20-06-41](https://user-images.githubusercontent.com/26736582/205455509-6e20b25f-2df4-4ed6-a05a-1cddb85287a8.png)